azerothcore/azerothcore-wotlk
# fix(DB/SAI): Uniting the Shattered Amulet

- Port missing target conditions for spell Fel Curse (12938) from TrinityCore
- Implement missing invincibility to the quest mobs (must be removed by using the quest items)



## Changes Proposed:
-  Importing missing conditions from TC.
-  Creatures should be invincible until the player uses the quest item to remove it.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7767
- Closes https://github.com/chromiecraft/chromiecraft/issues/1644

## SOURCE:

"Using the weapon (or salve) significantly reduces the abilities of the target, and enables you to actually defeat the mobs. Without it, you will be unable to reduce the mob past 1%. Equipping the weapon gives a 30 second cooldown to the on use ability, so equip it before combat."
https://wowpedia.fandom.com/wiki/Uniting_the_Shattered_Amulet

## Tests Performed:

- Tested in-game.

## How to Test the Changes:


1. npc add temp 7665/7666/7667
2. .damage 90000
3. Check that the mob doesn't die
4. .additem 10698
5. Use the item
6.  Kill the mob

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
